api/rhsm-subscriptions-api-spec.yaml

```diff
@@ -654,6 +654,33 @@ components:
           schema:
             $ref: "#/components/schemas/Errors"
   schemas:
+    GranularityGenerated:
+      type: string
+      enum: [ Daily, Weekly, Monthly, Quarterly, Yearly ]
+      x-enum-varnames: [ DAILY, WEEKLY, MONTHLY, QUARTERLY, YEARLY ]
+    ServiceLevelGenerated:
+      type: string
+      enum: [ "", Premium, Standard, Self-Support, _ANY ]
```

Comment:
The empty string displays like this...not a fan, but I might being too picky?

![image](https://user-images.githubusercontent.com/19955562/101080190-74fbbc80-3576-11eb-9227-ca5b5f03921d.png)

Reply:
I'm okay with it.  Might be worth filing an issue with the SwaggerUI team though.  Or fix it, submit a PR to them, ????, profit.